tide/wall/MovieUpdater.cpp

```diff
@@ -46,6 +46,36 @@
 #include "network/WallToWallChannel.h"
 #include "scene/MovieContent.h"
 
+namespace
+{
+void _splitSideBySide(const FFMPEGPicture& image, PicturePtr& left,
+                      PicturePtr& right)
+{
+    const auto width = image.getWidth() / 2;
+    const auto height = image.getHeight();
+    const auto format = image.getFormat();
+
+    left = std::make_shared<FFMPEGPicture>(width, height, format);
+    right = std::make_shared<FFMPEGPicture>(width, height, format);
+
+    const uint numTextures = (format == TextureFormat::rgba) ? 1 : 3;
+    for (uint texture = 0; texture < numTextures; ++texture)
+    {
+        const auto uvSize = image.getTextureSize(texture);
+        const size_t lineWidth = uvSize.width();
+        const size_t targetWidth = lineWidth / 2;
+        for (size_t y = 0; y < size_t(uvSize.height()); ++y)
```

Comment:
yes, I haven't benchmarked it but it didn't seem to be slow. Having to split the image is unfortunate but I see no other option with a side-by-side input format. There is no computation going on (except for the line indices) just memcpy for the lines, which should be mostly a memory rather than compute bottleneck, but we should vtune the application at some point anyways :-)